query/common/src/test/groovy/org/codice/ddf/admin/common/fields/base/BaseObjectFieldTest.groovy

```diff
@@ -118,4 +118,25 @@ class BaseObjectFieldTest extends Specification {
         then:
         subFieldOfInnerField.getValue() == 'valueChange'
     }
+
```

Comment:
Yeah, it's more for just an insurance for when this gets into other's hands and the code changes over time.